dot/rpc/modules/author_test.go

```diff
@@ -30,7 +32,7 @@ func TestAuthorModule_HasSessionKey_WhenScaleDataEmptyOrNil(t *testing.T) {
 
 	coremockapi := new(apimocks.CoreAPI)
 
-	decodeSessionKeysMock := coremockapi.On("DecodeSessionKeys", mock.AnythingOfType("[]uint8"))
+	decodeSessionKeysMock := coremockapi.On("DecodeSessionKeys", common.MustHexToBytes("0x0400"))
 	decodeSessionKeysMock.Run(func(args mock.Arguments) {
 		b := args.Get(0).([]byte)
 		dec, err := runtimeInstance.DecodeSessionKeys(b)
```

Comment:
Do we really need to instantiate a `*wasmer.Instance` to run this test?  Seems like this is the only thing slowing down this package at this point.